### ISSUE TITLE
DOMException: Failed to execute 'open' on 'XMLHttpRequest': Invalid URL

### ISSUE THREAD

**Issue by CoenraadS** [kleisauke/wasm-vips#35]
Hi, I'm really excited to try this project, but couldn't seem to get it running locally (Firefox & Chrome)

This was my minimum attempt:

```
<!DOCTYPE html>
<html>

<head>
    <title>wasm-vips</title>
</head>

<body>
</body>

<script src="./vips.js"></script>
<script type="module">
    const vips = await Vips();
</script>

</html>
```

Folder structure:
```
libvips
 ┣ index.html
 ┣ vips-jxl.wasm
 ┣ vips.js
 ┣ vips.wasm
 ┗ vips.worker.js
```

Console output:

```
Uncaught (in promise) DOMException: Failed to execute 'open' on 'XMLHttpRequest': Invalid URL
    at oa (http://127.0.0.1:5555/vips.js:12:500)
    at http://127.0.0.1:5555/vips.js:116:217
    at new Promise (<anonymous>)
    at e (http://127.0.0.1:5555/vips.js:116:191)
    at f (http://127.0.0.1:5555/vips.js:116:467)
    at fd (http://127.0.0.1:5555/vips.js:117:361)
    at http://127.0.0.1:5555/vips.js:118:327

worker sent an error! http://127.0.0.1:5555/vips.worker.js:1: Uncaught SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': Invalid URL
Lb.a.onerror @ vips.js:101
```

[project.zip](https://github.com/kleisauke/wasm-vips/files/10210944/project.zip)

**Comment by kleisauke** [comment]
Fixed with commit 5a21668ffaa48e6ea0b3756b5c4b2934f57f568d. This will be in v0.0.5.

**Comment by kleisauke** [comment]
v0.0.5 now available.
